models/users/index.js

```diff
@@ -3,11 +3,11 @@ const mongoose = require('mongoose');
 const userSchema = mongoose.Schema({
   name: {
     type: String,
-    require: true
+    required: true
```

Comment:
Makes sense to validate this in the controller functions because of them being required if not provided the schema will throw an error.